### PR TITLE
Clean Alluxio shell warnings

### PR DESCRIPTION
**Pull request by ssz1997**
Clean up some warnings in Alluxio shell

### CODE REVIEW

File: shell/src/main/java/alluxio/cli/LogLevel.java [hunk 1]
```diff
@@ -274,8 +274,7 @@ private static String parseOptLevel(CommandLine cmd) {
     return null;
   }
 
-  private static void setLogLevel(final TargetInfo targetInfo, String logName, String level,
-                                  AlluxioConfiguration alluxioConf)
+  private static void setLogLevel(final TargetInfo targetInfo, String logName, String level)
```
Comment (ssz1997) [inline shell/src/main/java/alluxio/cli/LogLevel.java:277]: alluxioConf is never used

File: shell/src/main/java/alluxio/cli/TestRunner.java [hunk 2]
```diff
@@ -89,7 +89,7 @@ private TestRunner() {} // prevent instantiation
    */
   public static void main(String[] args) throws Exception {
     TestRunner runner = new TestRunner();
-    JCommander jCommander = new JCommander(runner, args);
```
[the remark this answers is not in the document]
Comment (LuQQiu) [reply shell/src/main/java/alluxio/cli/TestRunner.java:92]: what's the null in middle, is null acceptable?

[the remark this answers is not in the document]
Comment (ssz1997) [reply shell/src/main/java/alluxio/cli/TestRunner.java:92]: It's `ResourceBundle`. I gave a second look and the comment on line 88 says no argument is needed, so I think it's better just get rid of the `args` as well. Then another constructor we are calling won't involve this confusing `null`.

**Review by apc999 — APPROVED** [review]

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**